dotcom-rendering/src/components/BigSixOnwardsContent.tsx

```diff
@@ -32,6 +32,10 @@ const containerStyles = css`
 	}
 `;
 
+const cardsContainer = css`
+	padding-top: ${space[2]}px;
```

Comment:
Only on large screen sizes do the lines overlap, but this padding exists for all screen sizes on the similarly-designed fronts containers, so I have done similar here.